libs/ui/src/lib/graphql-select/graphql-select.component.ts

```diff
@@ -331,7 +331,13 @@ export class GraphQLSelectComponent
     // this way we can wait for 0.5s before sending an update
     this.searchControl.valueChanges
       .pipe(debounceTime(500), distinctUntilChanged())
+      .pipe(takeUntil(this.destroy$))
```

Comment:
@vladi2998 
this has a quite bigger impact than just fixing the issue we have with the users dropdown because this is widely used

Reply:
this change thus, about the destroy, is indeed good so I'll keep it